pavlo976/BuildMinder-App
# Feature/employees statistics

### What's changed?
Employee statistics were created
### How to test it?
Check that the data is correct. Click the arrows to see if the carousel is moving correctly. Check if dragging is working
### What's missing?
Everything is up to date
